test/fixtures/mockRepositoriesData.json

```diff
@@ -54,11 +54,29 @@
 						"viewerPermission": "ADMIN"
 					},
 					{
-						"name": "media-upload-app",
-						"nameWithOwner": "name/media-upload-app",
+						"name": "responsive-design",
+						"nameWithOwner": "name/responsive-design",
+						"isPrivate": false,
+						"defaultBranchRef": {
+							"name": "master"
+						},
+						"viewerPermission": "ADMIN"
+					},
+					{
+						"name": "ecma262",
+						"nameWithOwner": "name/ecma262",
+						"isPrivate": false,
+						"defaultBranchRef": {
+							"name": "main"
+						},
+						"viewerPermission": "ADMIN"
+					},
+					{
+						"name": "agendas",
```

Comment:
none of these seem to have a `codeOfConduct` property?

Reply:
{
						"name": "ecma262",
						"nameWithOwner": "name/ecma262",
						"isPrivate": false,
						"defaultBranchRef": {
							"name": "main"
						},
						"viewerPermission": "ADMIN"
					},
					{
						"name": "agendas",
						"nameWithOwner": "name/agendas",
						"isPrivate": false,
						"defaultBranchRef": {
							"name": "main"
						},
						"viewerPermission": "ADMIN"
					} 



I added those @ljharb

Reply:
ecma262 this one has and agendas dont

Reply:
sure - but we won't be talking to the actual API, so this file is the only source of data. What about the data you added says that one has a CoC?

Reply:
@ljharb  I don't understand.If some of this repos will not have coc it will be printed as --- right?

Reply:
Yes. But the tests we want are to cover when it *has* a CoC also.